iron-list.html

```diff
@@ -395,7 +396,7 @@
       this._firstVisibleIndexVal = null;
 
       // random access
-      if (this._scrollBottom > this._physicalBottom || scrollTop < this._physicalTop) {
+      if (Math.abs(this._physicalBottom - this._scrollBottom) > this._physicalSize) {

```

Comment:
thanks for the explanation. I think you are right, I tested `if (Math.abs(delta) > this._physicalSize)` against a scroll perf tracker and it performs well. I think it's way more balanced and simpler. :)